src/libs/AxelarQueryAPI.ts

```diff
@@ -189,15 +191,23 @@ export class AxelarQueryAPI {
 
     const destTxFee = parseEther(gas_price).mul(gasLimit);
 
-    if (gasMultiplier > 1) {
-      return destTxFee
-        .add(baseFee)
-        .mul(gasMultiplier * 10000)
-        .div(10000)
-        .toString();
+    const calculatedFee =
+      gasMultiplier > 1
+        ? destTxFee
+            .add(baseFee)
+            .mul(gasMultiplier * 10000)
+            .div(10000)
+        : destTxFee.add(baseFee);
+
+    let finalFee = calculatedFee;
+
+    if (minGasFee) {
+      finalFee = calculatedFee.sub(BigNumber.from(minGasFee)).gt(0)
```

Comment:
Instead of comparing with the final fee, `minGasFee` should be compared with `gas_price`, then multiply with `gasLimit` at the prior step.

Reply:
updated in PR